ground/src/main/java/com/google/android/ground/model/job/Job.kt

```diff
@@ -19,21 +19,28 @@ import android.graphics.Color
 import com.google.android.ground.model.task.Task
 import timber.log.Timber
 
-/**
- * @param suggestLoiTaskType the type of task used to suggest the LOI for this Job. Null if the job
- * is already associated with an LOI.
- */
 data class Job(
   val id: String,
   val style: Style? = null,
   val name: String? = null,
   val tasks: Map<String, Task> = mapOf(),
-  val suggestLoiTaskType: Task.Type? = null,
+  val strategy: DataCollectionStrategy = DataCollectionStrategy.MIXED
 ) {
+  enum class DataCollectionStrategy {
+    PREDEFINED,
```

Comment:
So far, it's been helpful to have `UNKNOWN` for cases in when the web app evolves before the android app/field names change. Not sure if you want to add one here.

Reply:
I think UNKNOWN is useful for serialized representations like protos, but for model objects less so; if the strategy is invalid in remote or local data store, by the time to deserialize we should either have chosen a fall-back value, or thrown an error.